### ISSUE TITLE
[Question] Longhorn gets into bad state and cannot uninstall 

### ISSUE THREAD

**Issue by astachowiak** [longhorn/longhorn#2124]
**Question**

**Environment:**
 - Longhorn version: 1.1.0
 - Kubernetes version: K3S v1.19.5+k3s2
 - Node config
   - OS type and version Ubuntu 20.04
   - CPU per node: 
   - Memory per node: 8GB / 16GB
   - Disk type workers 1TB NVME
   - Network bandwidth and latency between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): Baremetal

I'm trying to install longhorn on my baremetal cluster running K3S. All works fine when I do that via helm install but I'm trying to setup deployment via flux v2. My configuration is:
master: 8GB, 32GB MMC
2 x worker: 16GB, 32GB MMC + 1TB NVME

I have tagged worker nodes with the nvme folders on system. All seems to work OK. But the longhorn-manager ends up in a bad state. I can't event remove longhorn anymore from the system (removal hangs).

The log states that each manager tries to update node resources but fails due to wrong resouceVersion I believe. What could be the cause of the issue?
[longhorn-manager-pm7ks.txt](https://github.com/longhorn/longhorn/files/5719652/longhorn-manager-pm7ks.txt)
[longhorn-manager-v7h98.txt](https://github.com/longhorn/longhorn/files/5719653/longhorn-manager-v7h98.txt)
[longhorn-manager-z22zg.txt](https://github.com/longhorn/longhorn/files/5719654/longhorn-manager-z22zg.txt)

**Comment by joshimoo** [comment]
The resource version conflict is a temporary state and should pass after the reque (reprocessing of the updated object).
What issue are you running into?

**Comment by astachowiak** [comment]
Hi,

the problem I have is that:

1. UI doesn't return anything - it loads but there is not data.
2. FluxCD thinks that the system was not deployed correctly
3. If I try to remove the longhorn-system namespace removal hangs

I have made some more tests and indeed it looks like the system itself works fine. I can create a PVC and PV via kubectl. The logs seems fine there. I see in the longhorn-manager-* that new items are picked up and created. I checked the storageclass and it is configures as per my annotations to nodes. But when I open the UI there is nothing. There is just a message (multiple) undefined: error. In the network console I see plenty of websocket errors: Error during WebSocket handshake: Unexpected response code: 200. Those errors are also then present in the longhorn-manager-* logs:

`time="2020-12-19T18:55:20Z" level=debug msg="websocket: close 1006 (abnormal closure): unexpected EOF" id=6073398925477503972 type=engineimages
10.42.2.1 - - [19/Dec/2020:10:39:26 +0000] "GET /v1/ws/1s/engineimages HTTP/1.1" 200 0 "" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"`

In the end it looks like something is not deployed correctly. But I can't seem to find the cause.
